<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMiliTwoStateSelect</class>
 <widget class="QWidget" name="CMiliTwoStateSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>250</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/common/mili1.png);
</string>
   </property>
   <widget class="QLabel" name="mRgMsbNoHint">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>179</y>
      <width>30</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">10</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelHint_1">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>179</y>
      <width>46</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelHint_2">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>118</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs2MsbHint">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>122</y>
      <width>46</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">23</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelHint_4">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>65</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">p0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelHint_5">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>65</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">p1</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1MsbHint">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>65</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">p9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mYDimHint">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>167</y>
      <width>46</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">23</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRgVal">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>205</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">ABh</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mYVal">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>145</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0xABCDEFG</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1Val">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>89</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs2SVal">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>114</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">ABh</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs2YVal">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>114</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">ABCDEFh</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMS1P0Val">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>40</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMS1P1Val">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>40</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMS1PnVal">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>40</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMS2S1Val">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>33</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1Ah</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMS2S0Val">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>47</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1Ah</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMS2Y1Val">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>48</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1ABCDEFh</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMS2Y0Val">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>32</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1ABCDEFh</string>
    </property>
   </widget>
   <widget class="QLabel" name="mSVal">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>226</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">ABh</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1SelVal">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>69</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">ABh</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
